mautic/mautic
# Protect Sent Emails from Public Access

| Q | A |
| --- | --- |
| Bug fix? | Yes |
| New feature? |  |
| Related user documentation PR URL | N/A |
| Related developer documentation PR URL | N/A |
| Issues addressed (#s or URLs) | #2678 |
| BC breaks? | No |
| Deprecations? |  |
#### Description:

Protect access to emails sent to users by executing a simple permissions check.
#### Steps to test this PR:
1. Browse to a contact who has had emails sent to them.
2. Click on an email for a public view, e.g. path /email/view/{hashId}
3. Copy that link to another browser where you are not logged in. You should receive a 404 error.
